src/new.js

```diff
@@ -178,6 +178,44 @@ export const iff = (ifFcn, ...rest) => {
 };
 
 /**
+ * Hook that executes a set of hooks and returns true if at least one of
+ * the hooks returns a truthy value and false if none of them do.
+ *
+ * @param {Array.function} rest - Hook functions to execute.
+ * @returns {Boolean}
+ *
+ * Example 1
+ * service.before({
+ *   create: hooks.some(hook1, hook2, ...) // same as [hook1, hook2, ...]
+ * });
+ *
+ * Example 2 - called within a custom hook function
+ * function (hook) {
+ *   ...
+ *   return hooks.some(hook1, hook2, ...).call(this, currentHook)
+ *     .then(hook => { ... });
+ * }
+ */
+
+export const some = (...rest) => (hook) => {

```

Comment:
I think the inner arrow function `(hook) => {` should be `function(hook) {` so the right context is given for `fn.call(this, ...)` later on